xbmc/epg/GUIEPGGridContainer.h

```diff
@@ -41,6 +38,8 @@ namespace EPG
 
   class CGUIEPGGridContainer : public IGUIContainer
   {
+    static constexpr size_t MAXBLOCKS = (33 * 24 * 60 / 5); //! 33 days of 5 minute blocks (31 days for upcoming data + 1 day for past data + 1 day for fillers)
```

Comment:
shouldn't need static here as the constexpr will basically be a "define" during compilation